services/nvidia-gpu-operator/23.3.1/defaults/cm.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nvidia-gpu-operator-23.3.1-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    nfd:
+        enabled: false
```

Comment:
hmm, it looks like automation is using 4 spaces indentation? we generally use 2 in our yaml files, wondering if we could use 2 here